### PR TITLE
Develop pmp revamp

### PR DESCRIPTION
**Pull request by Sohandey**

### CODE REVIEW

File: db_upgrade_scripts/mosip_pms/sql/1.3.0-DP.1_to_1.3.0_rollback.sql
Comment (mohanachandran-s) [inline db_upgrade_scripts/mosip_pms/sql/1.3.0-DP.1_to_1.3.0_rollback.sql:1]: Why this file is deleted?

File: db_upgrade_scripts/mosip_pms/sql/1.3.0-DP.1_to_1.3.0_upgrade.sql
Comment (mohanachandran-s) [inline db_upgrade_scripts/mosip_pms/sql/1.3.0-DP.1_to_1.3.0_upgrade.sql:1]: Why this file is deleted?

File: et --soft d2d5be7e
Comment (mohanachandran-s) [inline et --soft d2d5be7e:1]: Why this file is added?

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

**Review by mayuradesh — CHANGES_REQUESTED** [review]
change the PR description and add proper JIRA id and description

**Pull request closed without merging**